docs/internal_api.yml

```diff
@@ -58,6 +58,51 @@ paths:
           description: Internal server error.
           schema:
            $ref: "#/definitions/Error"
+  /devices/{id}/status:
+    put:
+      summary: Update the admission status of a selected device
+      description: |
+        Changes the given device's admission status.
+        The only possible status transition using this endpoint is from 'preauthorized' to 'accepted'.
+        Valid state transitions:
+        - 'preauthorized' -> 'accepted'
+      parameters:
+        - name: id
+          in: path
+          description: Device authentication data set identifier.
+          required: true
+          type: string
+        - name: status
+          in: body
+          description: New status
+          required: true
+          schema:
+            $ref: '#/definitions/Status'
+      responses:
+        200:
+          description: The status of the device authentication data set was successfully updated.
+          schema:
+            $ref: "#/definitions/Status"
```

Comment:
necessary? I'd get rid of it, unless it has a purpose.

Reply:
the only purpose here is to have it consistent with existing endpoints:
https://github.com/mendersoftware/deviceadm/blob/ff53ed5c55fa72f08e9316f12524a37ad23c0122/docs/management_api.yml#L278

Reply:
ack